### ISSUE TITLE
Ambiguous implicit in build.sbt for unusedCompileDependenciesFilter

### ISSUE THREAD

**Issue by alterationx10** [zio/zio-schema#89]
On a fresh pull of this project, I was getting an error from sbt while trying to load the project:

```
  [success] Total time: 6 s, completed Jul 9, 2021, 4:32:48 PM
  /Users/mrudolph/Projects/alterationx10/zio-schema/build.sbt:52: error: No implicit for Remove.Value[explicitdeps.ModuleFilter, sbt.librarymanagement.ModuleFilter] found,
  so sbt.librarymanagement.ModuleFilter cannot be removed from explicitdeps.ModuleFilter
  unusedCompileDependenciesFilter -= moduleFilter("org.scala-js", "scalajs-library")
  ^
  sbt.compiler.EvalException: Type error in expression
  [error] sbt.compiler.EvalException: Type error in expression
  [error] Use 'last' for the full log.
```

I believe the issue if from the `sbt-explicit-dependencies` plugin, along with something that's bringing in the sbt-updates plugin as well: see corresponding issue: https://github.com/cb372/sbt-explicit-dependencies/issues/33

The general rundown is:
> Summary: Ambiguous implicit import from same code in two projects...
> Solution: Explicit import of ambiguous implicit.

PR incoming...

**Comment by alterationx10** [comment]
To follow up, I cloned this on another computer, and it worked just fine.

After digging a bit deeper, I can see that I have `sbt-updates` set up in my global plugins (thanks to the autogenerated `project/.bloop/zio-schema-build.json` for helping me find that 🤣 ), so that explains it.

The explicit import could help other people who might have this plugin in scope (globally) like me, but 50/50 on the PR being worth merging 🤷

**Comment by thinkharderdev** [comment]
Fixed by #157
